### PR TITLE
Mid part

### PR DESCRIPTION
**Pull request by tafaramafemba**
The following additions were made to the project:
1. The works section was added.
2. The about me section was added.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,13 +32,144 @@ <h2 class="occupation">I’am a Software Developer</h2>
...
+                <p class="multiple-p">A daily selection of privately <br> personalized reads; no accounts or
+                    <br>sign-ups required.
+                    has been the industry's standard</p>
+                <ul class="languages">
+                    <li><button class="btn">html</button></li>
+                    <li><button class="btn boot">bootstrap</button></li>
+                    <li><button class="btn">Ruby</button></li>
+                </ul>
+                <button class="see pro">See Project</button>
+            </div>
```
Comment (RolandM99) [inline index.html:140]: - [ ] Please,  Add the button styling (Hover, enable, Active, Pressed ). Kindly, use this style for all your buttons.
Figma design :  
image

File: index.html [hunk 2]
```diff
@@ -29,13 +32,144 @@ <h2 class="occupation">I’am a Software Developer</h2>
...
+
+    </section>
+    <section class="about-me">
+        <h1 class="about">About me</h1>
+        <p class="services">Hello I’m a software developer! I can help you build a product , feature or website 
+            Look through some of my work and experience! If you like what you see and have a project you need
+            coded, don’t hestiate to contact me.</p>
+        <button class="btnresume">Get My Resume</button>
+        <img src="Images/illustration-about-me.png" class="img-me">
+
```
Comment (RolandM99) [inline index.html]: - [ ] Good work so far by implementing the design similar to the design provided by the Figma template. Make sure that you added the image illustration for the About page as a background image and not in the `img` tag. Please, let's rocking, fix it 😄

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @tafaramafemba , 

Good job so far! 🍾 
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯  
- No Linter errors ✔️ 
- Follow Github flow ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] [ OPTIONAL ] Good job for implementing a good README, but to make it more descriptive and professional use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) by adding the sections which missing in your README.
- [ ] Your Commit messages are not meaningful. Please, in the future make sure that your commit messages are meaningful. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (tafaramafemba) [reply index.html]: Thank you for this suggestion. I learnt something new by implementing this!

**Review by CollinsTatang — APPROVED** [review]
Hi @tafaramafemba , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**